doc/installing.rst

```diff
@@ -0,0 +1,19 @@
+..
+    :copyright: Copyright (c) 2022 ftrack
+
+.. _installing:
+
+**********
+Installing
+**********
+
+.. highlight:: bash
+
+Install the DCC framework plugins through the plugin manager within :term:`Connect` 2.
```

Comment:
@henriknorin-ftrack you can link the plugin manager documentation to close the loop
https://github.com/ftrackhq/ftrack-connect-plugin-manager

Reply:
Adding glossary for _plugin manager_, with intersphinx link